arm-automation/2015-10-31/swagger/account.json

```diff
@@ -142,6 +145,9 @@
             "in": "path",
             "required": true,
             "type": "string",
+			"minLength": 6,
+			"maxLength": 50,
+			"pattern": "^[A-Za-z][-A-Za-z0-9]{4,48}[A-Za-z0-9]$",
```

Comment:
Most of our parameter validations are done in b/e web service. The reason for this is, at times we have requirements from customer to increase Max length of parameter etc., for such cases its easy to fix and deploy b/e is production. Customer needs not update the SDK for changes. For this particular case ARM validates location and return appropriate localized error if location is incorrect.